_docs/1.0.0/usage-errcol.md

```diff
@@ -9,4 +9,43 @@ categories:
 Error Column
 ============
 
-`errCol` description will come here
+<!-- toc -->
+- [Description](#description)
+- [Error Types](#error-types)
+- [Notes](#notes)
+<!-- tocstop -->
+
+Description
+-----------
+
+`errCol` is a special, automatically created, composite column, that contains descriptions of all the issues encountered
+during **Standardization** and **Conformance** of the particular row.
+
+It's an array, where each member represents one error that happened on the particular row during its processing.
+The array member is a structured information itself:
+
+* `errType` - string representation of the type of the error
+* `errCode` - code representation of the type of the error in the form of _E#####_, where # is a digit (e.g. E00001)
+* `errMsg` - description of the error
+* `errCol` - name of the column, in which the error happened [\*](#notes-star)
+* `rawValues` - the input values for the error
+* `mappings` - ???
+
+
+Error Types
+-----------
+
+| Error type      | Description |
+|-----------------|-------------|
+| `stdCastError`  | When column value fails to standardize to the expected type |
+| `stdNullErr`    | When column value is `null` in non-nullable column during standardization |
+| `confMappingErr`| Mapping of the value failed during conformance |
```

Comment:
This is the odd one out but as far as descriptions go, it fits the purpose of describing the error (not the occasion) better. Could you remove the `When... ` from the other descriptions and change them to past tense as well?